dusty/command_file.py

```diff
@@ -0,0 +1,129 @@
+import os
+from nose.tools import nottest
```

Comment:
Importing test stuff in the main code is a no-no. I fixed this problem somewhere else before, lemme go find it